### DISCUSSION TITLE
Difference between magit fetch (f u) and git pull?

### DISCUSSION THREAD

**Discussion by tvraman** [magit/magit#4680] [Support] [answered]
At least when working with the Emacs Git repo etc, I notice the
following:

the state of the log buffer after (f u) in magit is different from
what I see when I do git pull from the commandline, then examine the
log with magit (l l)

Feature or bug? Also what is the magit equivalent of git pull?

**Comment by kyleam** [comment, marked as answer]
`f u` is running `git fetch $remote`.  To run `git pull ...`, you can use the `magit-pull` transient, bound to a capital `F` by default.

https://magit.vc/manual/magit/Pulling.html
